Feliz/Properties.fs

```diff
@@ -1453,3 +1454,14 @@ module prop =
         static member inline url = Interop.mkAttr "type" "url"
         /// Defines a week and year control (no timezone)
         static member inline week = Interop.mkAttr "type" "week"
+
+    [<Erase>]
+    type autoCapitalize =
+        /// No autocapitalization is applied (all letters default to lowercase)
+        static member inline off = Interop.mkAttr "autoCapitalize" "off"
```

Comment:
All of these should be lowercase: `autocapitalize`

Reply:
I thought so too at first, but then I saw how other props in this file use the same casing and it works as expected. Such as [this property](https://github.com/Shmew/Feliz/blob/patch-1/Feliz/Properties.fs#L349). I tested it out and it does apply the attribute in all lower case so I figured I should follow the style of the rest of the file.

Reply:
Oh, right. As long as it works :)